README.md

```diff
@@ -76,16 +75,6 @@ flowchart LR
     direction LR
 
 
-    subgraph commongrants["CommonGrants"]
-    direction LR
-        HHS/simpler-grants-protocol#299["Improvements - Quad 4"]
```

Comment:
These dropped out because I added a filter so we're only displaying co-planning proposals here, since that's what the section refers to